libs/labelbox/tests/integration/schema/test_user_group.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import faker
+from labelbox import Client
+from labelbox.schema.user_group import UserGroup, UserGroupColor, UserGroupUser, UserGroupProject
+
+data = faker.Faker()
+
+
+@pytest.fixture
+def client(client):
+    client.enable_experimental = True
+    client.endpoint = "https://app.lb-stage.xyz/api/_gql/"
```

Comment:
why hardcoding to stage?
Also, similar to Sam's comments. we could have an experimental_client fixture next to the client fixture in libs/labelbox/tests/conftest.py

Reply:
This is temporary, just a way to deal with  internal gql, I do plan to remove

Reply:
Resolve and leave sam's comment comment